anagrafica/models.py

```diff
@@ -1864,7 +1866,9 @@ def richiedi(self):
         self.autorizzazione_richiedi_sede_riferimento(
             self.persona,
             INCARICO_GESTIONE_TRASFERIMENTI,
-            invia_notifica_presidente=True
+            invia_notifica_presidente=True,
+            auto=Autorizzazione.AP_AUTO,
+            scadenza=settings.AUTORIZZAZIONE_AUTOMATICA,
```

Comment:
@AlfioEmanueleFresta intendevo qualcosa tipo::

    class Trasferimento(...):
        APPROVAZIONE_AUTOMATICA = timedelta(days=settings.DEFAULT_APPROVAZIONE_AUTOMATICA)
    ...
    self.autorizzazione_richiedi_sede_trasferimento(...
        scadenza=Trasferimento.APPROVAZIONE_AUTOMATICA, ...

E  analogamente tutte le classi figlie. Senza impostare nulla in ``ConAutorizzazioni``